defaults/main.yml

```
@@ -5,6 +5,9 @@ bbb_upgrade_packages: "{{ 'yes' if bbb_state == 'latest' else 'no' }}"
 bbb_check_for_running_meetings: true
 bbb_letsencrypt_enable: true
 bbb_letsencrypt_api: https://acme-v02.api.letsencrypt.org/directory
+bbb_nginx_install_from_source: true
```

@Davka, @BrutalBirdie should we change default source for nginx?
@DerDanilo The variable name `bbb_nginx_install_from_source` is a bit misleading, in my opinion.
```suggestion
bbb_nginx_install_from_official_repo: true
```